R/tbl_listing.R

```diff
@@ -0,0 +1,171 @@
+#' Create listings from a data frame
+#'
+#' This function creates a listing from a data frame, highlighting key columns. Common uses
+#' rely on few pre-processing steps, such as ensuring unique values in key columns or split
+#' by rows or columns. They are described in the note section.
+#'
+#' @param data (`data.frame`)\cr
+#'   a data frame containing the data to be displayed in the listing.
+#' @param keys ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   columns to be highlighted on the left of the listing.
+#' @param order_by ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   columns to be sorted. It defaults to `keys`.
+#' @param blank_rows_by ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
```

Comment:
FYI in the most recent draft of JL's guidence doc, blank rows are discouraged. Let's discuss whether to include this.

Reply:
ok let's keep it